### PR TITLE
Wizard cleanup

### PR DESCRIPTION
**Pull request by Spreeuw**
Start the wizard from the admin notice in a clean installation, or in an existing installation via `wp-admin/admin.php?page=wpo-wcpdf-setup`

### CODE REVIEW

**Review by alexmigf — COMMENTED** [review]
Missing escaping here:

https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/blob/dfba9c85d29793667e31a019a712b56cfd62c372/includes/views/setup-wizard/shop-name.php#L3-L4

File: includes/class-wcpdf-setup-wizard.php [hunk 1]
```diff
@@ -152,16 +148,16 @@ public function setup_wizard_steps() {
 		// array_shift( $output_steps );
 		?>
 		<div class="wpo-setup-card">
-			<h1 class="wpo-plugin-title">PDF Invoices & Packing Slips</h1>
+			<h1 class="wpo-plugin-title"><?php esc_html_e( 'PDF Invoices & Packing Slips' ); ?></h1>
```
[the remark this answers is not in the document]
Comment (Spreeuw) [reply includes/class-wcpdf-setup-wizard.php]: 3dfeb74

**Comment by alexmigf** [comment]
Miss escaping here?

https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/blob/dfba9c85d29793667e31a019a712b56cfd62c372/includes/class-wcpdf-settings-callbacks.php#L27

https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/blob/dfba9c85d29793667e31a019a712b56cfd62c372/includes/class-wcpdf-settings-callbacks.php#L36

https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/blob/dfba9c85d29793667e31a019a712b56cfd62c372/includes/class-wcpdf-settings-callbacks.php#L386

https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/blob/dfba9c85d29793667e31a019a712b56cfd62c372/includes/class-wcpdf-settings-callbacks.php#L423

File: includes/class-wcpdf-setup-wizard.php [hunk 2]
```diff
@@ -181,18 +177,18 @@ public function setup_wizard_content() {
 	 */
 	public function setup_wizard_footer() {
 		?>
-						<input type="hidden" name="wpo_wcpdf_step" value="<?php echo $this->step ?>">
+						<input type="hidden" name="wpo_wcpdf_step" value="<?php echo esc_attr( $this->step ); ?>">
 						<div class="wpo-setup-buttons">
-							<?php if ($step = $this->get_step(-1)): ?>
-								<a href="<?php echo $this->get_step_link($step); ?>" class="wpo-button-previous"><?php _e( 'Previous', 'woocommerce-pdf-invoices-packing-slips' ); ?></a>
+							<?php if ( $step = $this->get_step( -1 ) ): ?>
+								<a href="<?php echo $this->get_step_link( $step ); ?>" class="wpo-button-previous"><?php esc_html_e( 'Previous', 'woocommerce-pdf-invoices-packing-slips' ); ?></a>
```
[the remark this answers is not in the document]
Comment (Spreeuw) [reply includes/class-wcpdf-setup-wizard.php]: 9bb1cb5

**Comment by Spreeuw** [comment]
> Missing escaping here:
> 
> https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/blob/dfba9c85d29793667e31a019a712b56cfd62c372/includes/views/setup-wizard/shop-name.php#L3-L4

Fixed in 6d045fe

**Comment by alexmigf** [comment]
Captura de ecrã de 2021-11-30 12-51-17

* It's suppose to have that black border?
* The remove image is not working, not sure if it's related but I'm seeing this error: `Uncaught ReferenceError: addLoadEvent is not defined`

**Comment by Spreeuw** [comment]
> It's suppose to have that black border?

I didn't change those styles, so I guess it does? I'll ask @Terminator-Barbapapa to review the wizard styles again, it's possible that some of the core WP styles changed and affected the wizard too

> The remove image is not working, not sure if it's related but I'm seeing this error: `Uncaught ReferenceError: addLoadEvent is not defined`

I don't see that error, but it's not related to that button not working. Fixed in e258e73

**Review by alexmigf — APPROVED** [review]

**Pull request merged**